### DISCUSSION TITLE
How to extract the "most replayed" meta info?

### DISCUSSION THREAD

**Discussion by lofti198** [Tyrrrz/YoutubeExplode#769] [Q&A]
How to get this information about how often people replay the particular piece of the video?
![image](https://github.com/Tyrrrz/YoutubeExplode/assets/42960488/d2024597-ad79-4ea8-9588-981760decdd4)

**Comment by Tyrrrz** [comment]
This is not possible in YTE

**Reply by lofti198** [reply]
Thank you a lot for reply anyway. Maybe you could suggest the direction where I could proceed in searching for this feature?

**Reply by Tyrrrz** [reply]
I don't know, sorry
